upgrade.yml

```diff
@@ -104,6 +122,16 @@
       tags: ['infra', 'percona']
   environment: "{{ env_vars|default({}) }}"
 
+# post upgrade mysql will be behind haproxy. Set endpoint
+# so db_sync commands will work until post upgrade
+- name: set mysql connection string port 3306
```

Comment:
This task is actually telling the clients to use 3306 for the connection string. If you omit the port like that it assumes 3306. If we don't set it this way our plays will run and change the connection string to 3307 (which is why we set it back to 3307 at the end of upgrade). I can put the port in the variable it that makes it more obvious, otherwise i'm not sure what else to name this task.